region/lowernorfair/east.json

```diff
@@ -2610,7 +2609,41 @@
           "to": [
             {
               "id": 4,
-              "requires": [ "canKillRedKiHunters" ],
+              "requires": [ 
+                {"or": [
+                  {"and": [
+                    "heatProof",
+                    "canKillRedKiHunters"
```

Comment:
Same thought here as Red KiHunters shaft. If we can be expected to heatrun here without the means to kill them, it should probably be expected when heatproof as well.

Reply:
Roger, roger